docs/Bridging_Assets.md

```diff
@@ -0,0 +1,158 @@
+---
+id: Bridging_Assets
+title: Bridging Assets
+sidebar_label: qqq
+---
+
+The Arbitrum protocol's ability to [pass messages between L1 and L2](L1_L2_Messages.md) can be leveraged to trustlessly move assets from Ethereum to an Arbitrum chain and back. Any asset / asset type can in principle be bridged, including Ether, ERC20 tokens, ERC-721 tokens, etc.
+
+## Depositing And Withdrawing Ether
+
+To move Ether from Ethereum onto the Arbitrum chain, you execute a deposit transaction via `Inbox.depositEth`. This transfers funds to the Inbox contract on the L1 side and credits the same funds to you inside the Arbitrum chain at the specified address.
+
+```sol
+function depositEth(address destAddr) external payable override returns (uint256)
+```
+
+As far as Ethereum knows, all deposited funds are held by Arbitrum's Inbox contract.
+
+Withdrawing ether can be done using the [ArbSys](ArbSys.md) withdrawEth method:
+
+```sol
+`ArbSys(100).withdrawEth(destAddress)`
+```
+
+Upon withdrawing, the Ether balance is burned on the Arbitrum side, and will later be made available on the Ethereum side.
+
+`ArbSys.withdrawEth` is actually a convenience function is which is equivalent to calling `ArbSys.sendTxToL1` with empty calldataForL1. Like any other `sendTxToL1` cal, it will require an additional call to `Outbox.executeTransaction` after the dispute period elapses for the user to finalize claiming their funds on L1 (see ["L2 to L1 Messages Lifecycle && API"](L1_L2_Messages.md)). Once the withdrawal is executed from the Outbox, the user's Ether balance will be credited on L1.
+
+## Bridging ERC20 Tokens
+
+### Overview
+
+The Arbitrum protocol itself technically has no native notion of any token standards, and gives no pride-of-place to any particular token bridge. Described here is (effectively) a DApp with contracts on both Ethereum and Arbitrum that we refer to as our "Canonical Token Bridge"; it leverages Arbitrum's cross-chain message passing system to achieve basic desired token-bridging functionality. We recommend that you use it!
```

Comment:
Instead of "pride-of-place" maybe we should say it gives no built-in advantage to any token bridge. We do want to imply that most people should use our token bridge most of the time.  (Using a single bridge avoids the same L1 token living at multiple addresses on L2.)